native/src/widget/scrollable.rs

```diff
@@ -390,36 +461,39 @@ pub fn update<Message>(
     ) -> event::Status,
 ) -> event::Status {
     let bounds = layout.bounds();
-    let is_mouse_over = bounds.contains(cursor_position);
+    let mouse_over_scrollable = bounds.contains(cursor_position);
 
     let content = layout.children().next().unwrap();
     let content_bounds = content.bounds();
 
-    let scrollbar = scrollbar(
-        state,
+    state.create_scrollbars_maybe(
+        horizontal,
         scrollbar_width,
         scrollbar_margin,
         scroller_width,
         bounds,
         content_bounds,
     );
```

Comment:
Let's avoid methods with hidden side-effects like this one!

Instead, we can create a new `Scrollbars` type and return it:

```rust
let scrollbars = scrollbars(...);
```